tidb-server/main.go

```diff
@@ -113,6 +114,7 @@ func main() {
 	cfg.StorePath = *storePath
 	cfg.SlowThreshold = *slowThreshold
 	cfg.QueryLogMaxlen = *queryLogMaxlen
+  cfg.ProxyProtocol = *proxyProtocol
```

Comment:
Sorry, I made a mistake when resolving conflicts using GitHub web UI. The indention is incorrect here. Please help fix it when you are working with this file. Thanks a lot.